bot/cogs/help.py

```diff
@@ -153,7 +152,7 @@ def __init__(self):
         Will return an instance of the `HelpQueryNotFound` exception with the error message and possible matches.
         """
         choices = await self.get_all_help_choices()
-        result = process.extractBests(string, choices, scorer=fuzz.ratio, score_cutoff=90)
+        result = process.extractBests(string, choices, scorer=fuzz.ratio, score_cutoff=80)
```

Comment:
60 seems to work a lot better.

```suggestion
        result = process.extractBests(string, choices, scorer=fuzz.ratio, score_cutoff=60)
```